asmcli/commands/create-mesh.sh

```diff
@@ -22,18 +22,46 @@ parse_cluster_args() {
   fi
 
   while [[ $# != 0 ]]; do
-    local CLUSTER; CLUSTER="${1}"
-    context_append "clustersInfo" "${CLUSTER//\// }"
+    if [ -e "$1" ]; then
+      local KCF; KCF="${1}"
+      context_append "kubeconfigFiles" "${KCF}"
+    else
+      local CLUSTER; CLUSTER="${1}"
+      context_append "clustersInfo" "${CLUSTER//\// }"
+    fi
     shift 1
   done
 }
 
 validate_cluster_args() {
+  local KCF
+  local PROJECT_ID
+  local CLUSTER_LOCATION
+  local CLUSTER_NAME
+  local CTX_CLUSTER
+  local GKE_CLUSTER_URI
+
   # validate fleet id is valid
   get_project_number
 
+  # flatten any kubeconfig files into cluster P/L/C
```

Comment:
Can you add "this is GCP-only and will need to be reworked for other platforms" in a comment here as well? Just to make it more obvious.

Reply:
Ack